### PR TITLE
Fixed Quickstart

### PR DESCRIPTION
**Pull request by gagdiez**
The quickstart were outdated since the new version of `create-near-app` came out.

### CODE REVIEW

**Comment by gagdiez** [comment]
@bucanero I think the link checker is failing because I added a link in the `sidebar` to a new page, and is checking for the page on `near.org` instead of `localhost`, but otherwise it should work

File: docs/2.develop/contracts/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+<summary> Failing tests? </summary>
+
+If the tests are failing, make sure that you are using `node v16` and the `toolchain v1.69` in `rust`. You can always use
+
+- `nvm use 16` to switch to `node v16`
+- `rustup default 1.68` to switch to `toolchain v1.69`
+
+</details>
+
+In the background these commands are calling the build tools for each language, and invoking the [Sandbox](/develop/testing/integration-test) tests from the `sandbox-ts/rs` directory.
```
Comment (bucanero) [inline docs/2.develop/contracts/quickstart.md:159]:
```suggestion
In the background, these commands are calling the build tools for each language and invoking the [Sandbox](../testing/integration.md) tests from the `sandbox-ts/rs` directory.
```

File: docs/2.develop/contracts/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+:::tip Sandbox
+Testing the contracts within a Sandbox allows us to understand how the contract will behave once deployed to the network, while having total control over the testing environment.
```
Comment (bucanero) [inline docs/2.develop/contracts/quickstart.md:162]:
```suggestion
Testing the contracts within a Sandbox allows you to understand how the contract will behave once deployed to the network while having total control over the testing environment.
```

File: docs/2.develop/contracts/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+**Congrats**! your contract now lives in the NEAR testnet network.
+
+---
+
+## Interacting with the Contract
+
+To interact with your deployed smart contract you can call its methods using the `near-cli` tool.
+
+#### Get Greeting
+The `get_greeting` method is a [`view`](./anatomy.md#public-methods) method, meaning it only reads from the contract's state, and thus can be call for **free**.
```
Comment (bucanero) [inline docs/2.develop/contracts/quickstart.md:241]:
```suggestion
The `get_greeting` method is a [`view`](./anatomy.md#public-methods) method, meaning it only reads from the contract's state, and thus can be called for **free**.
```

File: docs/2.develop/integrate/quickstart.md [hunk 4]
```diff
@@ -22,63 +28,137 @@ If you already have [Node.js](https://nodejs.org/en/download) installed, simply
...
-A new browser window will open automatically with the code, give it a minute and the frontend will pop-up (make sure the pop-up window is not blocked).
+Once the app starts you will see the landing page, rendering a navigation bar that allows users to login using their NEAR wallet, and two pathways:
 
+![img](/docs/assets/examples/hello-near-landing-page.png)
+*Landing page of Hello NEAR Gateway*
 
-| 🌐 JavaScript                                                                                      | 🦀 Rust                                                                                            |
-|----------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
-| <a href="https://gitpod.io/#https://github.com/near-examples/hello-near-js.git">Open in Gitpod</a> | <a href="https://gitpod.io/#https://github.com/near-examples/hello-near-rs.git">Open in Gitpod</a> |
+Go ahead and sign in with your NEAR account. If you don't have one, you will be able to create one in the moment.
```
Comment (bucanero) [inline docs/2.develop/integrate/quickstart.md:63]:
```suggestion
Go ahead and sign in with your NEAR account. If you don't have one, you can create one on the fly.
```

File: docs/2.develop/integrate/quickstart.md [hunk 5]
```diff
@@ -22,63 +28,137 @@ If you already have [Node.js](https://nodejs.org/en/download) installed, simply
...
-Once the app starts you will see the screen below. Now go ahead and sign in with your NEAR account. If you don't have one, you will be able to create one in the moment.
+Login if you haven't done it yet and you will see a simple form that allows you to store a greeting in the smart contract.
 
-![img](/docs/assets/examples/hello-near.png)
-*Frontend of Hello NEAR*
+<hr class="subsection" />
 
-Once logged in, change the greeting and see how our Hello NEAR app greets you!
+### Under the Hood
+The interactions with NEAR are done by using the `useWallet` hook to retrieve the `signedAccountId` property, and `viewMethod` and `callMethod` methods from our `wallet selector`.
```
Comment (bucanero) [inline docs/2.develop/integrate/quickstart.md:108]:
```suggestion
Interactions with NEAR are done using the `useWallet` hook to retrieve both the `viewMethod` and `callMethod` methods and the `signedAccountId` property from the `wallet selector`.
```

File: docs/2.develop/integrate/quickstart.md [hunk 6]
```diff
@@ -22,63 +28,137 @@ If you already have [Node.js](https://nodejs.org/en/download) installed, simply
...
 
+<Github url="https://github.com/near/create-near-app/blob/master/templates/frontend/next/src/app/hello-near/page.js" language="jsx" start="12" end="32" />
+
+On load, the first `useEffect` hook will call the `get_greeting` method from the contract, and set the `greeting` state to the result.
```
Comment (bucanero) [inline docs/2.develop/integrate/quickstart.md:112]:
```suggestion
On load, the first `useEffect` hook will call the contract's `get_greeting` method and set the `greeting` state to the result.
```

File: docs/2.develop/integrate/quickstart.md [hunk 7]
```diff
@@ -22,63 +28,137 @@ If you already have [Node.js](https://nodejs.org/en/download) installed, simply
...
+
+If the user is logged in, the `saveGreeting` method will call the `set_greeting` method from the contract, and set the `greeting` state to the result.
```
Comment (bucanero) [inline docs/2.develop/integrate/quickstart.md:114]:
```suggestion
If the user is logged in, the `saveGreeting` method will call the contract's `set_greeting` method and set the `greeting` state to the result.
```

**Review by bucanero — APPROVED** [review]

**Pull request merged**